src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -113,6 +119,14 @@ def _type(value):
     return params
 
 
+def enum_default(resource_type, enum_name, enum_val_name):
+    mod = get_versioned_models(resource_type, enum_name)
+    try:
+        return getattr(mod, enum_val_name).value
+    except AttributeError:
+        return None
```

Comment:
I'm still concerned about trying to debug a mysterious typo with all of these things being strings. If we were to issue a logger.error, how much spam would we get in --debug for this?